### PR TITLE
feat: APPS 2389 2396 2397 Add identifier_global, history, edition metadata fields

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to APPS-[2389](https://uclalibrary.atlassian.net/jira/software/c/projects/APPS/boards/12?assignee=712020%3Af4539cc0-90d3-4445-9de9-94d4149de1e2&selectedIssue=APPS-2389)-[2396](https://uclalibrary.atlassian.net/jira/software/c/projects/APPS/boards/12?assignee=712020%3Af4539cc0-90d3-4445-9de9-94d4149de1e2&selectedIssue=APPS-2396)-[2397](https://uclalibrary.atlassian.net/jira/software/c/projects/APPS/boards/12?assignee=712020%3Af4539cc0-90d3-4445-9de9-94d4149de1e2&selectedIssue=APPS-2397)

Add identifier-global, history, & edition metadata fields

Screenshot 2023-12-07 at 3 12 25 PM

---

Screenshot 2023-12-07 at 3 17 52 PM

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -135,16 +135,17 @@ def self.modified_field
     config.add_show_field 'contents_note_tesim'
     config.add_show_field 'dimensions_tesim'
     config.add_show_field 'director_tesim'
+    config.add_show_field 'edition_tesim', label: 'Edition'
     config.add_show_field 'editor_tesim'
-    config.add_show_field 'engraver_tesim'
```
Comment (pghorpade) [inline app/controllers/catalog_controller.rb:139]: @jendiamond why is this removed?

File: app/controllers/catalog_controller.rb [hunk 2]
```diff
@@ -135,16 +135,17 @@ def self.modified_field
     config.add_show_field 'contents_note_tesim'
     config.add_show_field 'dimensions_tesim'
     config.add_show_field 'director_tesim'
+    config.add_show_field 'edition_tesim', label: 'Edition'
```
Comment (pghorpade) [inline app/controllers/catalog_controller.rb:138]: edition is multiple, not searchable and string so it should be _ssm which means string, stored,multiple
tesim means text,stored,indexed(searchable),multiple

**Review by pghorpade — CHANGES_REQUESTED** [review]

File: app/controllers/catalog_controller.rb (hunk 1 above)
Comment (jendiamond) [reply app/controllers/catalog_controller.rb:139]: accidental removal

File: app/controllers/catalog_controller.rb [hunk 3]
```diff
@@ -135,16 +135,18 @@ def self.modified_field
...
     config.add_show_field 'extent_tesim'
     config.add_show_field 'finding_aid_url_ssm'
     config.add_show_field 'foliation_ssi', label: 'Foliation note'
     config.add_show_field 'format_book_tesim', label: 'Format'
-    config.add_show_field 'masthead_parameters_ssi'
     config.add_show_field 'funding_note_tesim'
     config.add_show_field 'genre_tesim'
+    config.add_show_field 'history_tesim', label: 'History'
     config.add_show_field 'host_tesim'
+    config.add_show_field 'identifier_global_tesim', label: 'Identifier'
```
Comment (sourcefilter) [inline app/controllers/catalog_controller.rb:149]: identifier should be _ssim (string, not English text)

File: app/models/ucla_metadata.rb [hunk 4]
```diff
@@ -289,6 +297,10 @@ module UclaMetadata
       index.as :displayable
     end
 
+    property :identifier_global, predicate: ::RDF::URI.intern('http://id.loc.gov/ontologies/bibframe/Identifier') do |index|
+      index.as :stored_searchable
```
Comment (sourcefilter) [inline app/models/ucla_metadata.rb:301]: :symbol

**Review by sourcefilter — CHANGES_REQUESTED** [review]
Identifier should be a :symbol / _ssim field. I commented on this in a couple locations but not necessarily everywhere it needs to be changed
